app/src/main/cpp/BrowserWorld.cpp

```diff
@@ -782,80 +789,81 @@ vrb::TransformPtr
 BrowserWorld::CreateSkyBox(const std::string& basePath) {
   vrb::TransformPtr transform = vrb::Transform::Create(m.create);
   transform->SetTransform(Matrix::Position(vrb::Vector(0.0f, 0.0f, 0.0f)));
+  LoadSkybox(transform, basePath);
+  return transform;
+}
 
-  LoadTask task = [basePath](CreationContextPtr& aContext) -> GroupPtr {
-    std::array<GLfloat, 24> cubeVertices{
-        -1.0f, 1.0f, 1.0f, // 0
-        -1.0f, -1.0f, 1.0f, // 1
-        1.0f, -1.0f, 1.0f, // 2
-        1.0f, 1.0f, 1.0f, // 3
-        -1.0f, 1.0f, -1.0f, // 4
-        -1.0f, -1.0f, -1.0f, // 5
-        1.0f, -1.0f, -1.0f, // 6
-        1.0f, 1.0f, -1.0f, // 7
-    };
-
-    std::array<GLushort, 24> cubeIndices{
-        0, 1, 2, 3,
-        3, 2, 6, 7,
-        7, 6, 5, 4,
-        4, 5, 1, 0,
-        0, 3, 7, 4,
-        1, 5, 6, 2
-    };
+void
+BrowserWorld::LoadSkybox(const vrb::TransformPtr transform, const std::string& basePath) {
```

Comment:
Looks like indentation got messed up here? The lambda function looks like it is using four space instead of two?